contrib/dbus-test-runner/template.py

```diff
@@ -0,0 +1,35 @@
+pkgname = "dbus-test-runner"
+pkgver = "19.04.0"
+pkgrel = 0
+build_style = "gnu_configure"
+configure_args = ["--disable-static"]
+make_cmd = "gmake"
+hostmakedepends = [
+    "autoconf",
+    "automake",
+    "gmake",
+    "intltool",
+    "libtool",
+    "pkgconf",
+]
+makedepends = [
+    "dbus",
+    "dbus-glib-devel",
+    "gettext-devel",
```

Comment:
gettext is not necessary either

Reply:
should i also patch for not using gettext? configure requires gettext-devel

Reply:
i guess leave it, but move it to hostmakedepends